python/generate-pypi-package.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Needs to be done beforehand:
+# docker run -e PLAT=manylinux2014_x86_64 -it quay.io/pypa/manylinux2014_x86_64
```

Comment:
I would **strongly** prefer that setup is run natively on whatever platform we are on currently - not going through Docker.

Reply:
Not a good idea if you want to deploy to pypi. You need static builds and compatible libs otherwise (libc and such). A containerized and properly configured build environment is a very good idea, saving everybody headaches. This is the reason for having PEP's for this, e..g PEP599.

Reply:
Ok - just so there is no doubt about this: I can put up a show and complain a bit left and right, but when it comes to the build system the ultimate authority is @akva2